### PR TITLE
Run CI on GitHub Actions

### PR DESCRIPTION
**Pull request by walro**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [10.x, 12.x, 14.x]
```
Comment (walro) [inline .github/workflows/ci.yml:14]: For Travis CI we also ran on "latest stable". I found https://github.com/actions/setup-node/issues/61#issuecomment-577548687 and it seems like you could use `*` for that, but it looks a bit strange and also I think it's enough that we just test against the currently supported LTS versions

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        node-version: ${{ matrix.node }}
+
+    - name: Cache Node.js modules
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
```
Comment (roback) [inline .github/workflows/ci.yml:32]: What's this line for?

Couldn't spot it among the examples in https://github.com/actions/cache/blob/9c77c9dbfc97cd17e1add865174d4622362960f2/examples.md#node---npm :)

Comment (walro) [reply .github/workflows/ci.yml:32]: Hehe, pondered the same thing before, this is from their example at https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs#example-caching-dependencies

I think it's a fallback if someone would happen to cache something without `-node-`, but I must admit I never fully understood it and then forgot about it

Comment (walro) [reply .github/workflows/ci.yml:32]: Perhaps we should just remove the line? It will likely confuse a third person later on too :D

Comment (roback) [reply .github/workflows/ci.yml:32]: You're probably right about the fallback, but as long as it's documented I'm willing to 🙈

File: .github/workflows/ci.yml (hunk 1 above)
Comment (roback) [reply .github/workflows/ci.yml:14]: Agreed, it's not obvious that `*` means the latest stable. At least not until its documented (https://github.com/actions/setup-node/pull/104) :)

I think this is fine.

**Review by roback — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (walro) [reply .github/workflows/ci.yml:32]: 🙈

**Pull request merged**